insights/specs/__init__.py

```diff
@@ -269,6 +269,8 @@ class Specs(SpecSet):
     initscript = RegistryPoint(multi_output=True)
     insights_client_conf = RegistryPoint(filterable=True)
     installed_rpms = RegistryPoint()
+    intersystems_iris_cpf = RegistryPoint()
+    intersystems_iris_messages_log = RegistryPoint(filterable=True)
```

Comment:
Seems to need to move the two specs to after the `interrupts`. Or rename them as `iris_*`?